src/App.tsx

```diff
@@ -14,6 +17,21 @@ function App() {
         onSubmit={() => console.log('submitted')}
       />
       <LongButton onClick={() => null}>Example</LongButton>
+      <ActionButton
```

Comment:
From now on, let's remove these components from the App.tsx file once we finish building them. That is, we surely need to import them here for now to see how they render, but let's not commit them here. We have to treat the develop branch like our production code.

Reply:
I AM THE BEST!